lib/valkyrie/persistence/fedora/persister.rb

```diff
@@ -129,5 +131,15 @@ def validate_lock_token(resource)
 
         raise Valkyrie::Persistence::StaleObjectError, resource.id.to_s unless current_lock_token.serialize == retrieved_lock_token.serialize
       end
+
+      def should_set_last_modified?(resource)
+        return unless resource.optimistic_locking_enabled?
```

Comment:
@escowles Is is ok to return nil here instead of false?

Reply:
I dithered about that — I don't think it'll make any practical difference, but it's probably better to return `false`.

Reply:
@escowles just a quick question, otherwise, this is good to merge.